tests/bundle/Functional/TrashTest.php

```diff
@@ -133,6 +133,25 @@ public function testDeleteTrashedItemFailsWith404()
         self::assertHttpResponseCodeEquals($response, 404);
     }
 
+    public function testTrashLocation(): void
```

Comment:
Can you add a test against non-integer path parts, since we perform an assertion against in the code?

Reply:
https://github.com/ibexa/rest/pull/115/commits/0c23769c8a8f24a5de914dc0d0de8c7c74cbbb0e resolved here, we will eventually get 404 because of the way route is defined